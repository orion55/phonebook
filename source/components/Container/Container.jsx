import React from 'react';
import {Toolbar} from 'material-ui/Toolbar';

const Container = (props) => {
    return (
        <Toolbar>
            <div className='container'>
                {props.children}
            </div>
        </Toolbar>
    );
};

export default Container;
