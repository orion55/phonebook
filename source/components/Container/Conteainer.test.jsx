import React from 'react';
import renderer from 'react-test-renderer';
import Container from './Container';

describe('<Container>', () => {
    it('Render', () => {
        const tree = renderer.create(
            <Container/>
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });

    it('Render with children', () => {
        const tree = renderer.create(
            <Container>
                <div>Some content</div>
            </Container>
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
});
