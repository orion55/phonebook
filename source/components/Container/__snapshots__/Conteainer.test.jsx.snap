exports[`<Container> Render 1`] = `
<div
  className="container" />
`;

exports[`<Container> Render with children 1`] = `
<div
  className="container">
  <div>
    Some content
  </div>
</div>
`;
